plugins/modules/sonic_route_maps.py

```diff
@@ -313,6 +313,15 @@
               - IPv4 next hop address to set into a matching route in the
               - dotted decimal format A.B.C.D
             type: str
+          ip_next_hop_options:
```

Comment:
After some discussion and further consideration on this point, I would like to recommend that adding the 'native' option for IPv4 next hops be done by changing the current "ip_next_hop" option into a dictionary (similar to what is already implemented for IPv6) and locating the 'native' option in that dictionary, along with the current 'str' option, with the key/name of 'address'.

This same change would also need be done in the argspec and in the route map "model" (in the "models" repo).

While this is a "breaking" change and will need to be deferred to the 3.0.0 release, it will make the module/argspec/model more symmetrical for the two address families and easier for customers to understand and use versus splitting the IPv4 "next hop" options into two different option groups.